@navikt/core/react/src/date/hooks/useDatepicker.tsx

```diff
@@ -59,8 +57,8 @@ export interface UseDatepickerOptions
    */
   allowTwoDigitYear?: boolean;
   /**
-   * Opens datepicker on input-focus
-   * @default true
+   * Will be removed in a future major-version
+   * @deprecated
    */
   openOnFocus?: boolean;
```

Comment:
Kan eventuelt også gjøre dette i tilfelle noen ikke får med seg at den er deprecated? (Brekker ikke siden ingen setter den til true i dag?)
```suggestion
  openOnFocus?: false;
```